raster/r.geomorphon/main.c

```diff
@@ -155,23 +159,23 @@ int main(int argc, char **argv)
 	par_multi_prefix->type = TYPE_STRING;
 	par_multi_prefix->description =
 	    _("Prefix for maps resulting from multiresolution approach");
-	par_multi_prefix->guisection = _("Multires");
+	par_multi_prefix->guisection = _(GUI_MULTIRES);
```

Comment:
I have no idea if the macros work with the translation function (`_(...)`), specifically with the extraction for strings for translations. Any ideas? @marisn @veroandreo @neteler

Reply:
It would be fine to drop that specific commit to avoid confusion.

Reply:
Now, I'm wondering if the rest of the strings in that array, namely descriptions, are translated. Sending it to `_()` is not enough AFAIK. It really needs to be in `_()` to be picked for the translation files. With that, yes, drop it, so we can merge right away, but it would be good if somebody with translation experience checks this.

Reply:
A gettext macro on a variable AFAIK won't work as it will try to literally extract the variable name (but not expand it).

Reply:
Thanks @neteler . Actually, even in the current code, the documentation might not be translated, so that's something to keep an eye on.